src/utils/article-for.js

```diff
@@ -0,0 +1,15 @@
+/*
+ * A simplified take on which article to use before a noun.
+ * "A" goes before words that begin with consonants.
+ * "An" goes before words that begin with vowels.
```

Comment:
There is a really interesting JS file that takes this way further: http://home.nerbonne.org/A-vs-An/. Seems like overkill for the limited usage here.